feat(quicksettings): show image version

Angel-Padilla opened the pull request:
The image version reported by the backend is shown at the bottom of the quick settings item list

<img width="485" height="485" alt="image" src="https://github.com/user-attachments/assets/5544c30b-a971-4575-abc9-1916a7ed7038" />

Angel-Padilla committed:
feat(quicksettings): show image version

The image version reported by the backend is shown at the bottom
of the quick settings item list

Mimoja commented:
Can we add an explainer text? "Build Date: ...."?

Mimoja commented on src/components/QuickSettings/QuickSettings.tsx:
```diff
@@ -451,61 +474,85 @@ export function QuickSettings(): JSX.Element {
-                </Styled.Option>
-              )
-            )}
+            {settings.map((option, index) => {
```
These switch/case statements dont feel super maintainable, but I dont expect them to grow indefinetly either so I am fine with it

Mimoja commented on src/styles/utils/mixins.ts:
```diff
@@ -227,6 +233,22 @@ export const MenuAnnotation = styled.span<{ $marginRigth?: string }>`
   margin-right: ${({ $marginRigth }) => $marginRigth || '0'};
 `;
 
+export const SWVersionMenuAnnotation = styled.span<{ $marginRigth?: string }>`
```
This could also be an item in relative position to the last item, placed with ::after

Angel-Padilla commented:
> Can we add an explainer text? "Build Date: ...."?

Leaving only the date-hour as shown was a design decision from Carlos, I had a "version" legend before and got it removed

Angel-Padilla committed:
chore(quicksettings): remove os status from settings option

Use an ::before pseudo class in the Styled.OptionsContainer element to render
the OsUpdate status.

This way, it does not affect the indexing and traversal of the options

optionPositionOutter and optionPositionInner now need to account for that extra
element outside of settings

Angel-Padilla committed:
chore(quicksettings): remove sw version from settings option

Use an ::after pseudo class in the Styled.OptionsContainer element to render
the OsUpdate status.

This way, it does not affect the indexing and traversal of the options

Angel-Padilla committed:
chore(quicksettings): remove unnecessary switch statements

Angel-Padilla committed:
fix(quicksettings): sw version positioning

AluUriel commented:
:tada: This PR is included in version 1.93.0 :tada:

The release is available on [GitHub release](https://github.com/MeticulousHome/meticulous-dial/releases/tag/v1.93.0)

Your **[semantic-release](https://github.com/semantic-release/semantic-release)** bot :package::rocket:
